resources/js/components/Listing/ListingView.vue

```diff
@@ -0,0 +1,130 @@
+                            @filter-changed="filterChanged"
+                            @search-changed="searchChanged"
+                            @saved="$refs.presets.setPreset($event)"
+                            @deleted="$refs.presets.refreshPresets()"
+                            @restore-preset="$refs.presets.viewPreset($event)"
+                            @reset="filtersReset"
+                        />
+                    </div>
+
+                    <div
+                        v-show="items.length === 0"
+                        class="p-3 text-center text-grey-50"
+                        v-text="__('No results')"
+                    />
+
+                    <data-list-table
+                        v-show="items.length"
+                        :allow-bulk-actions="false"
+                        :loading="loading"
+                        :reorderable="false"
+                        :sortable="true"
+                        :toggle-selection-on-row-click="true"
+                        :allow-column-picker="true"
+                        :column-preferences-key="preferencesKey('columns')"
+                        @sorted="sorted"
+                    >
+                        <template slot="cell-label" slot-scope="{ row }">
+                            <a
+                                :href="cp_url(listingConfig.editUrl + row.id)"
+                            >{{ row.label }}</a>
+                        </template>
+
+                        <template slot="actions" slot-scope="{ row, index }">
+                            <dropdown-list>
+                                <dropdown-item
+                                    :text="__('Edit')"
+                                    :redirect="cp_url(listingConfig.editUrl + row.id)"
+                                />
+
+                                <dropdown-item
+                                    :text="__('Delete')"
+                                    class="warning"
+                                    @click="confirmDeleteRow(row.id, index)"
+                                    v-if="listingConfig.deleteUrl !== undefined"
+                                />
+                            </dropdown-list>
+
+                            <confirmation-modal
+                                v-if="deletingRow !== false"
+                                :title="__('Delete')"
+                                :bodyText="__('Are you sure you want to delete this item?')"
+                                :buttonText="__('Delete')"
+                                :danger="true"
+                                @confirm="deleteRow(listingConfig.listingUrl)"
+                                @cancel="cancelDeleteRow"
+                            ></confirmation-modal>
+                        </template>
+                    </data-list-table>
+                </div>
+            </div>
+        </data-list>
+
+        <data-list-pagination
```

Comment:
Also not quite sure where to put this but I seem to get a pagination related page when I load up the listing page. Any ideas? 
![image](https://user-images.githubusercontent.com/19637309/124025548-798f3300-d9e8-11eb-96d4-acb40450d62d.png)

Reply:
Another thing here - I don't see a pagination bar at the bottom of the `ListingView` which I should have because I have several hundred items for that model.